ci-operator/step-registry/openshift-qe/ingress-perf/openshift-qe-ingress-perf-commands.sh

```diff
@@ -18,11 +18,14 @@ fi
 cat /etc/os-release
 
 if [ ${BAREMETAL} == "true" ]; then
+  mkdir ~/.ssh
+  cp /secret/jh_priv_ssh_key ~/.ssh/id_rsa
```

Comment:
it might make more sense to put the pubkey  in `/bm/`

https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/ingress-perf/openshift-qe-ingress-perf-ref.yaml#L13-L15

WDYT?

Reply:
done, the vault part is in place as well